.github/workflows/ci.yml

```diff
@@ -134,6 +134,19 @@ jobs:
           echo "run_parachain_test=$run_parachain_test" | tee -a $GITHUB_OUTPUT
           echo "run_tee_test=$run_tee_test" | tee -a $GITHUB_OUTPUT
 
+  format-ts:
```

Comment:
How about appending it directly to `fmt` job? 😂

Reply:
I was considering that but decided to have it separated. Would you like to have it merged ?

Reply:
Actually yes :) so that we have all-in-one place for format check.
It doesn't take too long to complete this step so it doesn't necessarily need a separate step/job

Reply:
Ok, i'll merge. 
I was misleaded by thought  that `fmt` is specific to cargo's formatter 🙃